srcpkgs/hashboot/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'hashboot'
+pkgname=hashboot
+version=0.9.6
+revision=1
+build_style=fetch
+short_desc="Check integrity of files in /boot, mbr and bios"
+maintainer="teldra <teldra@rotce.de>"
+license="Hugware"
+homepage="https://github.com/tastytea/hashboot"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f45805430d77ed935616217bd78dbc79b8ee9e1ff1ffe6536fe1664a7243ae05
+
+do_install() {
+	tar xf ${version}.tar.gz
+	cd ${pkgname}-${version}
```

Comment:
This two lines should be not needed when you don't set a `build_style` at all.

Reply:
I tried but sadly it does not work.